.trampolinerc

```diff
@@ -0,0 +1,54 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Template for .trampolinerc
```

Comment:
If it's a template, why is it in the root of the repo?

Reply:
Ah my mistake, I'll delete that line. https://github.com/GoogleCloudPlatform/docker-ci-helper expects the `.trampolinerc` to be in the root of the repository. This is for use in autosynth's kokoro runs and should not be treated as a template.